tests/test_provider.py

```diff
@@ -1,8 +1,11 @@
+import os
+from unittest.mock import patch
```

Comment:
You dont need to import patch, it is available on the `mocker` fixture that should already be installed

```
mocker.patch()
```

Reply:
I think we have to install `pytest-mocker` to do this ?